gems/aws-sdk-dax/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Unreleased Changes
 ------------------
 
+1.48.1 (2023-09-30)
+------------------
+
+* Documentation - Added product offering link in the client.
+
```

Comment:
Not sure if this is right, but the template requested I fill this out. Let me know if this isn't the right place for this and I'll update. Thank you!